### DISCUSSION TITLE
Support for DB2390V12Dialect

### DISCUSSION THREAD

**Discussion by dhermanns** [hibernate/hibernate-orm#4652] [Design Proposals]
Hi!

Right now there is only support for DB2390V8. We have made an extensions for 390V12 to support
sequences.

I already saw that discussion:
https://discourse.hibernate.org/t/which-dialect-to-use-for-db2-on-z-os/5651/2

But I think it would be nice not to invent this over and over again ;-) .
We could prepare a Pull-Request for it.

Is there a desire for such a contribution?

**Comment by beikov** [comment]
Hi, right now we don't test Hibernate against DB2 for z/OS yet, so any contribution with a reference to the documentation is very welcome. Please note though, that we need the contribution for main which is Hibernate 6.0 first and only then would accept a backport for this.

**Reply by dhermanns** [reply]
Ok - great. What did you mean by "reference to the documentation"?

**Reply by beikov** [reply]
A link to the DB2 for z/OS documentation so that we can see the specification of something that you want to improve.

**Reply by dhermanns** [reply]
Ah ok. No problem. We will see what we can do...

**Reply by dhermanns** [reply]
Sequences seem to work fine for us with Hibernate 6. But my colleague contributed another small change here:
https://github.com/hibernate/hibernate-orm/pull/4666

**Reply by beikov** [reply]
Great, thanks for the PR!
